src/source-control/GitSourceControl.ts

```diff
@@ -359,7 +363,16 @@ export class GitSourceControl implements SourceControl {
       );
       const repo = await nodegit.Repository.open(this.repoPath);
       // SEE: Could be any of the long lived branches?
```

Comment:
Is this comment still relevant? Not sure what it's saying ahah

Reply:
Yeah, lol, I don't know what it means either, removing this